src/events/client/guildCreate.ts

```diff
@@ -0,0 +1,34 @@
+import Eris from "eris";
+import _logger from "../../logger";
+import { state } from "../../kmq";
+import { sendMessage, EMBED_INFO_COLOR } from "../../helpers/discord_utils";
+const logger = _logger("guildCreate");
+
+export default async function guildCreateHandler(guild: Eris.Guild) {
+    const client = state.client;
+    logger.info(`New server joined: ${guild.id} with ${guild.memberCount} users`);
+    const kmqDebugChannel: Eris.TextChannel = <Eris.TextChannel>client.guilds.find((guild: Eris.Guild) => {
+        if (guild.name === "KMQ") {
```

Comment:
Can get rid of the `if` and just do a single line return on the condition. 
Compare guild.id instead of name

Reply:
Use `.get` instead of `.find`

Reply:
Add these to config instead of hardcode